source/retryable-writes/retryable-writes.rst

```diff
@@ -200,10 +200,13 @@ is not possible and drivers MUST raise the original retryable error. In both
 cases, the caller is able to infer that an attempt was made.
 
 If the retry attempt also fails, drivers MUST update their topology according to
-the SDAM spec (see: `Error Handling`_) and raise any ensuing error to the user.
-In this case, the error from the retry attempt is just as relevant as the
-original retryable error and the caller is still able to infer that an attempt
-was made.
+the SDAM spec (see: `Error Handling`_). If an error would not allow the caller
+to infer that an attempt was made (e.g. connection pool exception originating
+from the driver), the original error should be raised. If the retry failed due
+to another retryable error or some other error originating from the server, that
+error should be raised instead as the caller can infer that an attempt was made
+and the second error is likely more relevant (with respect to the current
+topology state).
```

Comment:
As requested by @ShaneHarvey in https://github.com/mongodb/specifications/pull/228#discussion_r147221465.